docs/translations/translations-in-FSE-templates.md

```diff
@@ -0,0 +1,121 @@
+# Translations in FSE templates
+
+To make the WooCommerce Blocks plugin inclusive, all user-facing strings should be translatable. Manaing [Translations in PHP files](docs/translations/translations-in-PHP-files.md) and [Translations in JS/TS files](docs/translations/translations-in-JS-TS-files.md) is rather simple, given that both languages are programming languages. FSE-templates, however, are plain HTML files? So, how can we handle translations in FSE templates?
+
```

Comment:
I'm not sure having a question leading format here makes sense for the type of content.

Reply:
Good point. I rewrote that part.